sim/warrior/shield_block.go

```diff
@@ -7,29 +7,37 @@ import (
 	"github.com/wowsims/sod/sim/core/stats"
 )
 
-// TODO: Classic Update
 func (warrior *Warrior) RegisterShieldBlockCD() {
 	actionID := core.ActionID{SpellID: 2565}
-	cooldownDur := time.Second * 60
+	cooldownDur := time.Second * 5
 
 	warrior.ShieldBlockAura = warrior.RegisterAura(core.Aura{
-		Label:    "Shield Block",
-		ActionID: actionID,
-		Duration: time.Second * 10,
+		Label:     "Shield Block",
+		ActionID:  actionID,
+		Duration:  time.Second * time.Duration(5+[]float64{0, 0.5, 1, 2}[warrior.Talents.ImprovedShieldBlock]),
+		MaxStacks: 1 + []int32{0, 1, 1, 1}[warrior.Talents.ImprovedShieldBlock],
```

Comment:
This could just be a `core.Ternary` and either 1 or 2 depending on if the talent is > 0